src/cargo/sources/registry/mod.rs

```diff
@@ -781,13 +779,9 @@ impl<'cfg> Source for RegistrySource<'cfg> {
                     }
                     any_pending |= self
                         .index
-                        .query_inner(
-                            name_permutation,
-                            &req,
-                            &mut *self.ops,
-                            &self.yanked_whitelist,
-                            f,
-                        )?
+                        .query_inner(name_permutation, &req, &mut *self.ops, &mut |s| {
+                            f(s.into_summary());
+                        })?
```

Comment:
Is there a reason this doesn't do filtering?

Reply:
This is in a `kind == QueryKind::Fuzzy` so it is only being used for looking for near misses for error messages. At this point, the more results we can return the better even if there yanked or similar.

Reply:
Should we put that in as a comment?